src/nrniv/netpar.cpp

```diff
@@ -79,18 +73,6 @@ extern double nrnmpi_step_wait_; // barrier at beginning of spike exchange.
  */
 extern "C" int nrnthread_all_spike_vectors_return(std::vector<double>& spiketvec, std::vector<int>& spikegidvec);
 
-// BGPDMA can be 0,1,2,3,6,7
-// (BGPDMA & 1) > 0 means multisend ISend allowed
```

Comment:
This is the multisend method implemented via MPI_ISend/MPI_IRecv that works on any machine. When this got copied over to CoreNEURON and some aspects re-implemented, BGPDMA got changed to NRN_MULTISEND

Reply:
Oh ok! Didn’t realised that as we don’t have CMake option for it.

Reply:
So endly we keep it, thanks for the insight